authorization_services/topics/policy-js-policy.adoc

```diff
@@ -42,28 +42,41 @@ if (contextAttributes.containsValue('kc.client.network.ip_address', '127.0.0.1')
 }
 ```
 
-You can also use role-based access control (RBAC):
+You can also use Role-Based Access Control (RBAC) in your policies. It is possible to check for
+any *realm* role granted to an user:
 
```

Comment:
You can also use Role-Based Access Control (RBAC) in your policies. Use this to determine which, if any, *realm* role has been granted to an user: